src/Console/Command/RollbackCommand.php

```diff
@@ -37,9 +31,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $oldVersion = $updater->getOldVersion();
 
             $output->writeln(sprintf('Rollback from version %s to %s!', $oldVersion, $newVersion));
+
             return 0;
         } catch (\Exception $e) {
-            $output->writeln('Error: ' . $e->getMessage());
+            $output->writeln('Error: '.$e->getMessage());
```

Comment:
Call to method `writeln` from undeclared class `\Symfony\Component\Console\Output\OutputInterface`